robotoff/prediction/ocr/origin.py

```diff
@@ -0,0 +1,264 @@
+        
+    if len(ingredients_origins) == 0:
+        return []
+
+    return [
+        Prediction(
+            type=PredictionType.nutrient,
+            data={"ingredients_origins": origin, "version": EXTRACTOR_VERSION},
+        )
+        for origin in ingredients_origins
+    ]
+
+def check_if_general_word_in_ingredients(general_words_regex: List[str], ingredients: str):
+    """
+    Utility function for checking if one of the general words like 'elements' or 'ingredients' is 
+    present in the list of ingredients returned by extract_ingredients_from_match_string
+    """
+    for reg in general_words_regex:
+        if re.search(reg, ingredients) is not None:
+            return True
+    return False
+
+def extract_origin_from_match (origin_match, lang: str) -> str:
+    if origin_match.group("in_or_outside") is not None and origin_match.group("in_or_outside") in OUTSIDE_WORDS_BY_LANG[lang]:
+        return LARGE_ORIGIN # the sentence tells that the product is made outside a zone, 
+        # so the only thing we can extract is that it has a large origin
+    elif origin_match.group("negation") is not None:
+        return LARGE_ORIGIN # the sentence tells that the product is not made in the specified zone, 
+        # so the only thing we can extract is that it has a large origin
+    elif origin_match.group("country") is not None:
+        def standardize (s):
+            return " ".join((word[0].upper() + word[1:].lower() for word in s.split(" ")))
+        standardized_name = standardize(origin_match.group("country"))
+        try:
+            return next( #find the translation in english of the specified country name 
+                country_id
+                for country_id, country_content in COUNTRIES_IN_ALL_LANGS.items()
+                if country_content["name"][lang] == standardized_name
+            )
+        except StopIteration: #impossible, if the regex matched with the group country, the country with this name exists
+            return standardized_name
+    elif origin_match.group("country_adj") is not None and origin_match.group("country") in COUNTRY_ADJECTIVE_TO_COUNTRY_NAME_BY_LANG[lang]:
+        return COUNTRY_ADJECTIVE_TO_COUNTRY_NAME_BY_LANG[lang][origin_match.group("country_adj").lower()]
+    elif origin_match.group("large_origin") is not None:
+        return LARGE_ORIGIN
+    else:
+        return UNKNOW_ORIGIN
+
+def extract_ingredients_from_match_string (ingredients_text: str, lang) -> List[str]:
+    """
+    Utility function that splits the regex's matched group "ingredient" into a list of ingredients.
+    the regex matches with sentence with several ingredients, like 'quinoa and rice are made in France'
+    It match the whole "quinoa and rice " string, and this function splits each ingredients.
+    """
+    splitted = ingredients_text.split(" ")
+    def is_not_empty (s):
+        return s != ""
+    splitted = filter(is_not_empty, splitted)
```

Comment:
Shouldn't you simply use split() as it is more powerful (split on multiple space and any type of spaces).
```
>>> "a   b  c \r\t   d \n\t".split()
['a', 'b', 'c', 'd']
```

```suggestion
    splitted = ingredients_text.split()
```